docs/src/main/mdoc/jump_start_guide.md

```diff
@@ -94,27 +94,6 @@ If the provided `option` is `Some(x)`, it becomes `Right(x)`.
 Otherwise it becomes `Left` with the provided `ifNone` value inside.
 
 
-
-# `instances` packages and `apply` syntax
-
-## `import cats.instances.<F>._`
-
-There are a couple of type classes that are fundamental to Cats (and generally to category-based functional programming), the most important ones being
-[`Functor`](http://typelevel.org/cats/typeclasses/functor.html),
-[`Applicative`](http://typelevel.org/cats/typeclasses/applicative.html) and
-[`Monad`](http://typelevel.org/cats/typeclasses/monad.html).
-We're not going into much detail in this blog post (see e.g. the [already mentioned tutorial](https://www.scala-exercises.org/cats/functor)),
-but what is important to know is that to use most Cats syntax, you also need to import the implicit type class instances for the structures you're operating with.
-
-Usually, it's just enough to import the appropriate `cats.instances` package.
-For example, when you're doing the transformations on futures, you'll need to `import cats.instances.future._`.
-The corresponding packages for options and lists are called `cats.instances.option._` and `cats.instances.list._`.
-They provide the implicit type class instances that Cats syntax needs to work properly.
-
-As a side note, if you have trouble finding the necessary `instances` or `syntax` package, the quick workaround is to just `import cats.implicits._`.
-This is not a preferred solution, though, as it can significantly increase compile times — especially if used in many files across the project.
-It is generally considered good practice to use narrow imports to take some of the implicit resolution burden off the compiler.
-
 ## `import cats.syntax.apply._`
```

Comment:
🤔 We may want to change the heading here, since this was styled after a-la-carte syntax imports

Reply:
Changed to ```import cats.syntax.all._```